example.rules

```diff
@@ -0,0 +1,9 @@
+# The count of CPUs per node, useful for getting CPU time as a percent of total.
+instance:node_cpu:count = count(node_cpu{mode="idle"}) without (cpu,mode)
+
+# CPU in use by CPU.
+instance_cpu:node_cpu_not_idle:rate2m = sum rate(node_cpu{mode!="idle"}[2m])) without (mode)
+
+# CPU in use by mode.
+instance_mode:node_cpu_not_idle:rate2m = sum rate(node_cpu{mode!="idle"}[2m])) without (cpu)
```

Comment:
Following that to its logical conclusion you'd have a rule per label value you wanted to exclude, which is obviously not sane.

If the user doesn't care about the idle time they can filter it out when requesting the metric.